src/lib/esnext.decorators.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference lib="es2015.symbol" />
+
+interface SymbolConstructor {
+    readonly metadata: unique symbol;
+}
```

Comment:
We should also add something like this so that the metadata is accessible:

```ts
interface Function {
  [Symbol.metadata]: DecoratorMetadata | null;
}
```